packages/webapp/src/components/ImagePicker/index.tsx

```diff
@@ -75,7 +75,7 @@ export default function ImagePicker({
       dropContainerRef.current?.classList.toggle(styles.dropContainerActive);
     } else if (e.type === 'drop') {
       const file = e.dataTransfer?.files[0];
-      if (file) showImage(file);
```

Comment:
Any chance of `file` not existing?

Reply:
It's weird because when I first wrote that, TypeScript was complaining that file is possibly undefined but now its not. So I don't think it's possible for file to be undefined but I'm not 100% sure.